backend/Cyberia/views.py

```diff
@@ -0,0 +1,102 @@
+from urllib import response
+from rest_framework.decorators import api_view
+from django.http import JsonResponse
+from dotenv import load_dotenv
+from rest_framework.response import Response
+from rest_framework import status
+from pathlib import Path
+import os
+import requests
+from Cyberia.NotionAPIparser import * 
+from threading import Thread
+
+dotenv_path = Path('backend/.env')
+load_dotenv(dotenv_path)
+token = os.getenv('token')
+version = os.getenv('version')
+components = dict()
+
+# Create your views here.
+
+@api_view(['Get'])
+def Cyberia(request):
```

Comment:
Nitpick: Can you change the function name to lowercase, to follow python naming conventions (https://www.python.org/dev/peps/pep-0008/)